AnomalyDetection/LibLinear/configs/liblinear-example.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--
+    Description:
+        An example configuration file for a one-class SVM optimized in the dual.
```

Comment:
Since we're editing comments anyway, I'm not sure what "optimized in the dual" is supposed to mean here.

Reply:
There are two ways of solving the linear SVM problem, the primal form which has variables per feature and the dual form which has variables per data point. The nu-SVM here only works in the dual form. The comment is probably irrelevant as it doesn't really add much information.